lib/checks/aria/aria-prohibited-attr-evaluate.js

```diff
@@ -27,14 +28,12 @@ import standards from '../../standards';
  */
 function ariaProhibitedAttrEvaluate(node, options, virtualNode) {
   const prohibited = [];
-  const role = getRole(virtualNode);
-
-  if (!role) {
-    return false;
-  }
 
+  const role = getRole(virtualNode);
   const attrs = virtualNode.attrNames;
-  const prohibitedAttrs = standards.ariaRoles[role].prohibitedAttrs;
+  const prohibitedAttrs = role
```

Comment:
Apologies for not thinking of this sooner, but I think this should check if this is a "null" attribute. Don't think this should complain about `<div aria-label=" " aria-labelledby="doesNotExist">`.